crypto/test/gtest_main.cc

```diff
@@ -43,33 +37,6 @@ int main(int argc, char **argv) {
     }
 #endif
 
-#if defined(TEST_ARM_CPUS)
```

Comment:
Related to my comment: I wonder if we wanted to keep this capability for non-fips buids.

Reply:
No need for that because we added the same tests using `all_tests.go` configs where we can set the capability variable by exporting an env var.